pkg/nextgen/model/model_test.go

```diff
@@ -0,0 +1,124 @@
+package model
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/json"
+	"github.com/nuts-foundation/nuts-crypto/test"
+	core "github.com/nuts-foundation/nuts-go-core"
+	"github.com/nuts-foundation/nuts-registry/pkg/cert"
+	"github.com/nuts-foundation/nuts-registry/pkg/types"
+	"github.com/stretchr/testify/assert"
+	"net/url"
+	"testing"
+	"time"
+)
+
+var vendorId, _ = core.NewPartyID("1.2.3.4", "vendorId")
+var organizationId, _ = core.NewPartyID("4.3.1.1", "organizationId")
+var endpointType, _ = NewOIDURN("1.2.3.4", "http")
+var location, _ = url.Parse("https://nuts.nl")
+var notBefore = time.Date(2020, 10, 20, 20, 30, 0, 0, time.UTC)
+
+func TestMarshalVendor(t *testing.T) {
```

Comment:
What are you testing in this lib? The go language or the json library?

Reply:
Conformance of the model to the spec (RFC006); whether the outputted JSON is like the spec has spec-ed it (correctness of tags and types).